packages/core/contracts/financial-templates/funding-rate-store/implementation/FundingRateStore.sol

```diff
@@ -1,8 +1,72 @@
 pragma solidity ^0.6.0;
 pragma experimental ABIEncoderV2;
 
+import "../interfaces/FundingRateStoreInterface.sol";
+import "../../../common/implementation/Testable.sol";
 
-// TODO:
-contract FundingRateStore {
 
+contract FundingRateStore is FundingRateStoreInterface, Testable {
+    struct Proposal {
+        FixedPoint.Signed rate;
+        uint256 time;
+        address proposer;
+    }
+
+    struct FundingRateRecord {
+        FixedPoint.Signed rate;
```

Comment:
Does this mean that the default `rate` is 0 for an identifier?

Reply:
Yep, that's correct.